src/stubs.ts

```diff
@@ -9,6 +9,7 @@ import {
   ConcreteComponent,
   ComponentPropsOptions
 } from 'vue'
+import type { Teleport } from 'vue'
```

Comment:
Instead of grabbing the type, grab the actual component then we can do the comparison (as commented below). Eg `import { Teleport } from 'vue'`.

Reply:
See my previous comment. I got this working so I'll remove it!